src/main/java/org/sonarsource/sonarlint/ls/DefaultClientInputFile.java

```diff
@@ -103,18 +104,18 @@ private static String toSqLanguage(@Nullable String clientLanguageId) {
       case "vue":
       case "vue component":
       case "babel es6 javascript":
-        return "js";
+        return Language.JS;
       case "python":
-        return "py";
+        return Language.PYTHON;
       case "typescript":
       case "typescriptreact":
-        return "ts";
+        return Language.TS;
       case "html":
-        return "web";
+        return Language.HTML;
       case "oraclesql":
-        return "plsql";
+        return Language.PLSQL;
       default:
-        return clientLanguageId;
+        return null;
```

Comment:
What about Java, PHP, APEX, ...? Either put everything in the switch case, or I think better is to use `Language.forKey(clientLanguageId).orElse(null)`